core/src/executor/evaluate/error.rs

```diff
@@ -57,4 +57,7 @@ pub enum EvaluateError {
 
     #[error("subquery returns more than one row")]
     MoreThanOneRowReturned,
+
+    #[error("function does not support following DataType: {0}")]
```

Comment:
It seems that the information about the `Format` function is missing from the display information.
So it seems to be a very broad error type.